### ISSUE TITLE
Broken html when chat with bot in 1-1 mode (private messages)

### ISSUE THREAD

**Issue by AndrewBurdyug** [skoczen/will#189]
Hi! 

Seems like a bot send all messages in plain text(in private chat) even if used reply/say methods with **html=True**.

You can reproduce this, just send to bot  a 'help' message in private chat, 
you will receive back help message with broken formatting.

**Comment by skoczen** [comment]
Hey @AndrewBurdyug!

Thanks for submitting an issue!

Yep, the behavior you describe is true.  It's because hipchat doesn't support html chat, so instead of dumping raw HTML in, we strip it out.  

(AFAIK.  If this has changed and there is now html 1-1 support, please let me know!)

**Comment by AndrewBurdyug** [comment]
Let me check this out...

**Comment by AndrewBurdyug** [comment]
https://www.hipchat.com/docs/apiv2/method/private_message_user (see message_format section)

**Comment by skoczen** [comment]
Interesting. That's new and exciting.  Wonder if it extends to XMPP messages, or if we'd have to route around to use the v2 api.

**Comment by AndrewBurdyug** [comment]
I see you already use HipChat v2 API in **mixins/hipchat.py** and I think you not need to worry about XMPP =)

I made experiment, I disabled **html_to_text** in **plugin.py**: 

```python 
    32          if kwargs.get("html", False) and (message and message['type'] in ('chat', 'normal')):
    33              # 1-1 can't have HTML.
    34              # content = html_to_text(content)
    35              pass
```

And I get HTML message from a bot in 1-1 dialog. 

So, I think you need made some changes in the 'if' condition(that's all) and all will be fine!

**Comment by skoczen** [comment]
Actually, I think you can just drop that if statement entirely - mind submitting a PR?  (Make sure to add yourself to `AUTHORS`!)

Thanks for checking in on this!

**Comment by AndrewBurdyug** [comment]
>I think you can just drop that if statement entirely

yes, it was made just for testing of html messages in 1-1 dialog with bot, 
I'll make PR!

**Comment by skoczen** [comment]
Fantastic, thanks!

**Comment by Regner** [comment]
This used to work. Something seems to have changed recently with the HipChat API or something. We just upgraded HipChat server from 1.3.1 to 1.3.4 and this broke. :(

**Comment by skoczen** [comment]
Hey @Regner, can you be more specific with "this broke" ?

**Comment by Regner** [comment]
Sorry my apologies.

![image](https://cloud.githubusercontent.com/assets/815037/10611905/db9ec7b8-773d-11e5-879b-292dcada3dbb.png)

The message at 11:20 is with HipChat server 1.3.1 and the message at 11:30 is with the newer version. The bot shouldn't have been changed at all in those times.

**Comment by brandonsturgeon** [comment]
Hipchat was kind enough to update the way they handle various `/` commands (eg: `/code` `/quote`) and HTML through the api. 

I know this is a very late response, but I remember running into the same wall at about this same time, and I came to the conclusion that there wasn't anything Will could do on their end. 

However, I did put in a ticket and was redirected to an already existing ticket:
https://jira.atlassian.com/browse/HCPUB-201

So it's a known issue, and I'm actually very surprised they haven't fixed it yet. I remember seeing some progress awhile ago.
